prelude/prelude.purs

```diff
@@ -547,6 +660,7 @@ module Prelude
 
   infixr 5 <>
 
+  -- | Concatenation-like operation.
```

Comment:
Re laws, I think this typeclass only has associativity: `(a <> b) <> c == a <> (b <> c)`